ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/WebServiceEngine.java

```diff
@@ -95,18 +99,25 @@ public void initGrizzlyHttpService(BundleContext bc) throws IOException {
         rc.registerClasses(RestServer.class, SseResource.class, SseFeature.class);
         rc.register(new ResponseFilter());
         restServer = GrizzlyHttpServerFactory.createHttpServer(ServerRepository.getInstance().getBaseUriREST(), rc);
-        // Configure document root at the ARE/data/webservice folder
-        restServer.getServerConfiguration().addHttpHandler(new StaticHttpHandler("./data/webservice"), "/");
+ 
+        // Normal Web server configuration (document root)
+        String docRoot=ResourceRegistry.getInstance().toString(ResourceRegistry.getInstance().getResource("/",RES_TYPE.WEB_DOCUMENT_ROOT));
```

Comment:
this seems to be not formatted. doesn't matter, but just as a reminder ;) Doesn't eclipse offer an option to auto-format on save or something like this?

Reply:
ok, I will search for autosaving ;-)